rbac_api.go

```diff
@@ -210,15 +210,13 @@ func (e *Enforcer) GetImplicitPermissionsForUser(user string, domain ...string)
 // GetImplicitUsersForPermission("data1", "read") will get: ["alice", "bob"].
 // Note: only users will be returned, roles (2nd arg in "g") will be excluded.
 func (e *Enforcer) GetImplicitUsersForPermission(permission ...string) ([]string, error) {
-	pSubjects := e.GetAllSubjects()
-	gInherit := e.model.GetValuesForFieldInPolicyAllTypes("g", 1)
-	gSubjects := e.model.GetValuesForFieldInPolicyAllTypes("g", 0)
+	subjects := e.GetAllSubjects()
```

Comment:
Don't change code in this CI PR.

Make a new code-specific PR if necessary.

Reply:
Sorry, I didn't sync upstream code.